### PR TITLE
first query takes more than default 500ms

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

**Comment by kkraune** [comment]
agreed. but it must work now, and the warmup is non-trivial

**Comment by kkraune** [comment]
some more comments:
* we can add something in the framework, but I don't think so as it is access logged and can be confusing (where did it come from?)
* the first query can be considered a warmup - so maybe just have that in the sample app as such, to explain the concept - I will add a comment on this

File: basic-search-java/src/test/java/com/mydomain/example/FeedAndSearchSystemTest.java [hunk 1]
```diff
@@ -38,7 +38,9 @@ void feedAndSearch() {
         var deleteResult = endpoint.send(endpoint.request(documentPath).DELETE());
         assertEquals(200, deleteResult.statusCode());
 
+        // the first query needs a higher timeout than the default 500ms, to warm up the code
         var emptyResult = endpoint.send(endpoint.request("/search/", Map.of("yql", yql, "timeout", "5s")));
```
Comment (jonmv) [inline basic-search-java/src/test/java/com/mydomain/example/FeedAndSearchSystemTest.java]: Ok, now it _is_ explicit warm-up code ;)

**Comment by jonmv** [comment]
Yes, my intention was just to fire off a warm-up query at the start of the test, with a higher timeout, and state explicitly it was for warm-up.

**Pull request merged**